### ISSUE TITLE
The Clear Button is not accessible via Tab key for keyboard users.

### ISSUE THREAD

**Issue by talatkuyuk** [Hacker0x01/react-datepicker#3884]
The problem is: The keyboard users can not focus to the **Clear Button** via `Tab` key.

Clear button's tab index (tabIndex) actually should be `0` instead of `-1`. Or, could be the same tabIndex with `this.props.tabIndex`.

https://github.com/Hacker0x01/react-datepicker/blob/master/src/index.jsx#L1069

**Comment by talatkuyuk** [comment]
The problem still exists even after 2 years...

**Comment by martijnrusschen** [comment]
Contributions are welcome

**Comment by talatkuyuk** [comment]
I will make a PR, setting tabIndex to 0 (zero) instead of -1. Is it okey?
